Makefile

```diff
@@ -161,6 +168,11 @@ firefox: bzip2 wget
 
 geckodriver: node-wct-local
 
+golangci-lint: curl gpg
+	if [ "$$(which golangci-lint)" == "" ]; then \
+		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/${GOLANGCI_LINT_VERSION}/install.sh | sh -s -- -b $$(go env GOPATH)/bin; \
```

Comment:
This command is failing in the CI checks

https://github.com/web-platform-tests/wpt.fyi/actions/runs/4991412664/jobs/8937860406#step:3:138